v7/libretto/README.md

```diff
@@ -0,0 +1,58 @@
+Sophisticated, simple, and classically styled, Libretto is designed to showcase
+your writing prowess! A unique feature of this theme is that it does NOT rely
+on any Javascript. It's built using HTML, CSS and a few fonts. Best of all,
+the theme is responsive - yes, it works on mobiles and tablets too!!
+
+
+# Theme Highlights
+
+* Classic looks!
+* Navigation links on top-right (for tablet/desktop)
+* Blog description at bottom
+* Support for custom copyright notices at bottom-left
+* Icon links to social sites at bottom-right
+* Mobile support! (Looks good on my android phone, at least!)
+* No Javascript! (for now..?)
+
+This is my first effort at making a theme and I hope you will enjoy using it!
+
+
+# Official DEMO website
+
+In addition to the demo hosted on themes.getnikola.com, I also maintain a demo
+site dedicated to showcasing the strengths of this theme here,
+
+<https://note2self.abraham-v.com/libretto-theme-for-nikola/>
+
+Speaking of which, the source code for this theme (and demo) is hosted on
+Github,
+
+<https://github.com/note2self-abrahamvarricatt/libretto-theme-for-nikola>
+
+Feel free to report any issues, make suggestions or even better - contribute!
+
+
+# Un-supported features
+
+Here is a list of Nikola features (that I'm aware of) which are currently not
+supported with this theme,
+
+* 3rd-party comment system  (is it even possible to do this without JS ?)
+* Tags
```

Comment:
The original theme didn't have any UI for it. I didn't want to 'improvise' too much. Already did that for the archives part.